cmd/create/operatorroles/by_prefix.go

```diff
@@ -20,6 +20,10 @@ import (
 	"github.com/openshift/rosa/pkg/rosa"
 )
 
+const (
+	validMinMinorVersion = "4.0"
```

Comment:
Is that the min minor version for creating operator roles by prefix?

Reply:
I just setup something that will always pass basically, because there's no min version. Maybe skip the validation within the function when it is empty instead?

Reply:
Changed to skip validation if supplied version is empty, what do you think?

Reply:
Ok